xdsl/transforms/loop_nest_lowering_utils.py

```diff
@@ -68,6 +62,105 @@ def indices_for_map(
     return output_indices
 
 
+def _insert_loop_nest(
+    rewriter: PatternRewriter,
+    insertion_point: InsertPoint,
+    zero_op: arith.Constant,
+    one_op: arith.Constant,
```

Comment:
These names look bit arbitrary, maybe can explain in comments or docs what they do?

Reply:
They're literally the one and zero constant for the bounds, I'll document this